code/optimize/src/androidTest/java/com/adobe/marketing/mobile/optimize/OptimizeFunctionalTests.java

```diff
@@ -535,57 +534,65 @@ public void testGetPropositions_decisionScopeInCache()
+                    + "              {\r\n"
+                    + "                \"trafficType\": \"0\",\r\n"
+                    + "                \"step\": \"entry\"\r\n"
+                    + "              },\r\n"
+                    + "              {\r\n"
+                    + "                \"trafficType\": \"0\",\r\n"
+                    + "                \"step\": \"display\"\r\n"
+                    + "              }\r\n"
+                    + "            ],\r\n"
+                    + "            \"correlationID\": \"716226:0:0\",\r\n"
+                    + "            \"decisionProvider\": \"TGT\",\r\n"
+                    + "            \"experience\": {\r\n"
+                    + "              \"id\": \"0\"\r\n"
+                    + "            }\r\n"
+                    + "          },\r\n"
+                    + "          \"scope\": \"someDecisionScope\",\r\n"
+                    + "          \"id\": \"aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa\",\r\n"
+                    + "          \"items\": [\r\n"
+                    + "            {\r\n"
+                    + "              \"schema\":"
+                    + " \"https://ns.adobe.com/personalization/default-content-item\",\r\n"
+                    + "              \"meta\": {\r\n"
+                    + "                \"activity.name\": \"Some Test Activity\",\r\n"
+                    + "                \"profile.timeNow\": \"1722212083855\",\r\n"
+                    + "                \"profile.audienceUserNeed\": \"\",\r\n"
+                    + "                \"profile.language\": \"\",\r\n"
+                    + "                \"experience.name\": \"Default Content\",\r\n"
+                    + "                \"profile.site\": \"\",\r\n"
+                    + "                \"profile.url\": \"\",\r\n"
+                    + "                \"profile.subjects\": \"\",\r\n"
+                    + "                \"profile.path\": \"\",\r\n"
+                    + "                \"profile.subjectPrimary\": \"\",\r\n"
+                    + "                \"profile.translatedTabbedShelfTitle\": \"Discover SBS in 63"
+                    + " Languages\",\r\n"
+                    + "                \"profile.environment\": \"production\",\r\n"
+                    + "                \"profile.brandName\": \"\",\r\n"
+                    + "                \"profile.audioChannelLastPlayed\": \"\",\r\n"
+                    + "                \"profile.type\": \"\"\r\n"
+                    + "              },\r\n"
+                    + "              \"id\": \"0\"\r\n"
+                    + "            }\r\n"
+                    + "          ]\r\n"
+                    + "        }\r\n"
+                    + "      ],\r\n"
+                    + "      \"requestId\": \"someRequestId\",\r\n"
+                    + "      \"requestEventId\": \""
                         + requestEventId
```

Comment:
why are we modifying this test instead of adding a new one?  is the existing test no longer valid?

Reply:
This test was failing post our changes that's why i've updated the test @sbenedicadb

Reply:
Your changes should not have any effect on this test. Please revert the full test to how it was originally and try running it again